cw-orch-daemon/src/senders/sign.rs

```diff
@@ -0,0 +1,145 @@
+use crate::{
+    cosmos_modules,
+    queriers::Node,
+    tx_broadcaster::{
+        account_sequence_strategy, assert_broadcast_code_cosm_response, insufficient_fee_strategy,
+        TxBroadcaster,
+    },
+    CosmTxResponse, DaemonError, QuerySender, TxBuilder, TxSender,
+};
+use cosmrs::{
+    proto::cosmos::authz::v1beta1::MsgExec,
+    tendermint::chain::Id,
+    tx::{Body, Fee, Raw, SignDoc, SignerInfo},
+    AccountId, Any,
+};
+use prost::Message;
+
+pub struct SigningAccount {
+    pub account_number: u64,
+    pub sequence: u64,
+}
+
+pub trait CosmosSigner: QuerySender<Error = DaemonError> + Sync {
+    // --- General information about the signer --- //
+    /// The chain id of the connected chain
+    fn chain_id(&self) -> String;
+
+    /// The account id of the signer.
+    /// This is `_` prefixed because this conflict wih the TxSender trat
+    fn _account_id(&self) -> AccountId;
```

Comment:
If it's a signer, it should also be a `TxSender` no? Can we add `TxSender` to the trait bound of this type to prevent this conflict?

Reply:
Or alternatively, remove `account_id` from `TxSender` and make `TxSender: CosmosSigner`

Reply:
If we don't rename it or move it then we can still call it `account_id`. 

If both traits are implemented the call just needs to specify the trait. 

`Signer::account_id`
`TxSender::account_id`